src/qml/qgismobileapp.qml

```diff
@@ -666,90 +665,105 @@ ApplicationWindow {
   }
 
 
-  Controls.Menu {
+  Menu {
     id: mainMenu
     title: qsTr( "Main Menu" )
 
-    Controls.MenuItem {
+    width: Math.max(200*dp, mainWindow.width/4)
+
+    MenuItem {
       id: openProjectMenuItem
       property ProjectSource __projectSource
 
+      font.pointSize: 14
+      width: parent.width
+      height: 48 * dp
+
       text: qsTr( "Open Project" )
-      iconSource: Style.getThemeIcon( "ic_map_white_24dp" )
       onTriggered: {
         __projectSource = platformUtilities.openProject()
+        highlighted = false
       }
     }
 
-    Controls.MenuSeparator {}
+    MenuSeparator { width: parent.width }
 
-    Controls.MenuItem {
+    MenuItem {
       text: qsTr( "Settings" )
 
+      font.pointSize: 14
+      width: parent.width
+      height: 48 * dp
+
       onTriggered: {
         qfieldSettings.visible = true
```

Comment:
@signedav , FYI, we need to close the dashBoard here now that it's a drawer, otherwise the Settings pop up appears below the drawer.